lib/src/utils/builders/attachment_builder.dart

```diff
@@ -62,7 +62,7 @@ class AttachmentBuilder {
   /// See: https://discord.com/developers/docs/reference#image-data
   String getBase64() {
     final encodedData = base64Encode(_bytes);
-    final extension = path_utils.extension(_name);
+    final extension = path_utils.extension(_name).substring(1);
```

Comment:
This will throw if the file has no exception. It might be a good idea to include a "sensible default" like PNG in case the image file has no extension

Reply:
Yeah - maybe extract `path_utils.extension(_name)` to a variable so we don't compute it twice but that's the idea. Even better - make `'png'` an optional named parameter named `defaultFormat`.